core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -993,6 +993,12 @@ class DAGScheduler(
           JavaUtils.bufferToArray(closureSerializer.serialize((stage.rdd, stage.func): AnyRef))
       }
 
+      if (taskBinaryBytes.length > TaskSetManager.TASK_SIZE_TO_WARN_KB * 1024) {
```

Comment:
The original code version seems to be warning about full task, including rdd, shuffle dep, closures, partition and so on.
I think that the previous warning code was retained because the #1498 did not completely modify it,
In most cases, the size of  serialized TaskDescription is smaller than tasks`s Broadcast.